scoring/tests/test_scoring.py

```diff
@@ -21,14 +21,15 @@ def construct_scorer(self, zone_contents):
 
     def assertScores(self, expected_scores, zone_contents):
         scorer = self.construct_scorer(zone_contents)
+        scorer.validate(None)
```

Comment:
This is a reasonable check to have here, though I'm not sure we should use this as the mechanism to check the validation behaviour. Doing so forces the validation tests to pass an expected scores value which is misleading as the value should be meaningless.

Reply:
[c825fff](https://github.com/srobo/sr2022-comp/pull/2/commits/c825fff2da46ff21205b1b61ee70d489211fd4e8)